vpr/src/route/rr_graph.cpp

```diff
@@ -1396,10 +1396,10 @@ static void build_rr_sinks_sources(const int i,
         return;
 
     auto type = grid[i][j].type;
-    int num_class = type->num_class;
-    t_class* class_inf = type->class_inf;
+    int num_class = (int)type->class_inf.size();
+    std::vector<t_class> class_inf = type->class_inf;
     int num_pins = type->num_pins;
-    int* pin_class = type->pin_class;
+    std::vector<int> pin_class = type->pin_class;
```

Comment:
Minor, but we don't need to copy these. Should be able to take as `const &`

Reply:
Done